stablehlo/transforms/StablehloLegalizeToVhlo.cpp

```diff
@@ -40,52 +40,45 @@ Attribute convertAttrToVhlo(Attribute stablehloAttr) {
   // Handle StableHLO attributes.
   // The logic that handles attributes from other dialects (e.g. builtin
   // attributes) lives below.
-  if (auto attr = stablehloAttr.dyn_cast<stablehlo::ChannelHandleAttr>()) {
+  if (auto attr = stablehloAttr.dyn_cast<stablehlo::ChannelHandleAttr>())
```

Comment:
Let's keep these for consistency within the file. Strange for some of the `if`s in a block to have braces while others don't.